packages/liveblocks-client/src/types/SerializedCrdt.ts

```diff
@@ -8,15 +8,23 @@ export enum CrdtType {
 }
 
 export type SerializedCrdt =
+  | SerializedRootObject
   | SerializedObject
   | SerializedList
   | SerializedMap
   | SerializedRegister;
 
+export type SerializedRootObject = {
+  type: CrdtType.OBJECT;
+  parentId?: undefined;
+  parentKey?: undefined;
+  data: JsonObject;
+};
```

Comment:
NOTE: This is almost the same as a "normal" SerializedObject. Please note that this is NOT a new Op code or something like that. Just codifies at the type level that either both keys are set, or both are not.